src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -91,9 +99,9 @@
    public void initializeUI(UIBuilder builder) throws Exception
    {
       UIContext context = builder.getUIContext();
+      Mission mission = (Mission) context.getAttributeMap().get(Mission.class);
```

Comment:
It may work in the web version, but if you install the addon and run inside JBDS for example, it will fail, since the command state is kept between page navigation